src/com/sap/piper/integration/TransportManagementService.groovy

```diff
@@ -116,9 +121,14 @@ class TransportManagementService implements Serializable {
         }
 
         def response = sendApiRequest(parameters)
+
+        if (config.verbose) {
+            echo("Received response '${response.content}' with status ${response.status}.")
+        }
+
         echo("Node upload successful.")
```

Comment:
Same answer. You are claiming about code which was not changed by the PR. Of course also here it would be better to be more precise. Here again I would like to suggest to provide that in another PR in order to keep the PR focused.

Reply:
Agree. I will prepare another PR, when this one is merged.